buildbot/configure.py

```diff
@@ -185,6 +185,7 @@ def do_configure(args):
         "-DSYCL_ENABLE_KERNEL_FUSION={}".format(sycl_enable_fusion),
         "-DSYCL_ENABLE_MAJOR_RELEASE_PREVIEW_LIB={}".format(sycl_preview_lib),
         "-DBUG_REPORT_URL=https://github.com/intel/llvm/issues",
+        "-DLLVM_EXPERIMENTAL_TARGETS_TO_BUILD=SPIRV",
```

Comment:
SPIR-V backend is developing to be a viable alternative to SPIR-V LLVM translator. We would like to start using/testing it in the SYCL offloading compilation pipeline. This will increase the build time and space a bit, but SPIR-V backend is a considerably 'small' backend and the benefits are worth the overhead. Adding @bader for visibility and comments if any. Thanks

Reply:
I suggest we pass this option to configure.py script from GitHub Actions script (e.g. https://github.com/intel/llvm/blob/sycl/.github/workflows/sycl-linux-build.yml#L31). This way it will be enabled in CI, but do not impact default configuration for developers.
When we can pass all CI tests with SPIR-V back-end, we can move the change from GHA script directly to configure.py.

Reply:
Hi @bader,

Thanks for the suggestion. I will take a look. A bigger question is this. In this PR, I am adding the flag '-fsycl-use-spirv-backend-for-spirv-gen' which is turned off by default. How do we want to run CI on this? Do we keep submitting a 'draft PR' once in a while with the option turned on? Or is there a better way? Thanks

Reply:
I think we can add it here. https://github.com/intel/llvm/blob/157310a2f6cd4c190d2d699546692e2619b564df/.github/workflows/sycl-linux-build.yml#L172

I will add this to the next commit. Thanks

Reply:
I am also adding this to the linux and windows builds. I am not adding this for macos and post-commit builds. Hope that's agreeable.

Thanks